.circleci/config.yml

```diff
@@ -498,8 +498,29 @@ jobs:
       - checkout:
           name: Checkout Streamlit code
 
-      - update-submodules
+      - run:
+          name: set envvar
```

Comment:
Supernit: For consistency I'd capitalize these, and maybe also rename as "Set preview branch envvar"

Also, I'd move this right above "Create Wheel File", on the general principle that code should be defined close to where it's used. This lets the code reader juggle fewer things in their head. (Eg it makes a future consolidation of our "checkout/pre-cache/..." CircleCI jobs easier to reason about)